src/assets/index.js

```diff
@@ -56,106 +58,108 @@ const analysis_report_json = {
 const update_result = (report) => {
     //clearing previous results, if any
     var tags = ["#symptoms", "#treatment", "#products"];
-    for(var i=0 ; i<tags.length;i++){
-      const del_list = document.querySelector(tags[i]);
-      del_list.innerHTML = '';
-    }
-    //adding current result
-    const show_disease = document.querySelector("#disease");
-    show_disease.textContent = report.Disease;
-
-    const show_symptoms = document.querySelector("#symptoms");
-    const symptoms = Object.values(report.Symptoms);
-    symptoms.forEach((item) => {
-        const list = document.createElement("li");
-        list.textContent = item;
-        document.querySelector("#symptoms").appendChild(list);
-    });
-
-    const treatment = Object.values(report.Treatment);
-    treatment.forEach((item) => {
-        const list = document.createElement("li");
-        list.textContent = item;
-        document.querySelector("#treatment").appendChild(list);
-    });
 
-    const product = Object.values(report.Recommended_Product);
-    product.forEach((item) => {
-        const med = item.split(":")[0].trim();
-        const url = `${item.split(":")[1].trim()}:${item.split(":")[2].trim()}`;
+    for (var i = 0; i < tags.length; i++) {
+        const del_list = document.querySelector(tags[i]);
+        del_list.innerHTML = '';
+
```

Comment:
hey, I think you removed the closing bracket the loop from here!
Be attentive, otherwise, it will be very hard to debug later!

Reply:
Overall work is great!

Reply:
Okay I'll correct this Sir!